### DISCUSSION TITLE
intro

### DISCUSSION THREAD

**Discussion by giscus** [ZoruaFox/Personal-Blog#1] [Announcements]
# intro

关于燕栖屿 茫茫海屿，有燕而居。 因为之前有群友想看我搞个博客，所以用VuePress勉强开了一个，但是更新可能不勤（） 会记录一些杂七杂八的东西，感觉到时候可以分好几个类，有点小小期待把一些零碎知识整合起来会是什么样子。

https://zorua.top/intro.html

**Comment by ZoruaFox** [comment]
Hello, world!

**Comment by ajdgg** [comment]
哈喽

**Comment by ajdgg** [comment]
如果在这里评论会不会同步

**Comment by yangyan-ot** [comment]
更新....emm？怎么就这一篇文章(？
